src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
check out the documentation of the `nextInt()` method. The bound parameter there is exclusive, which means max number that we can get here is 99.
Also, make this number a constant